### PR TITLE
Refactor seeds and tests

### PR DESCRIPTION
**Pull request by roschaefer**

### CODE REVIEW

File: src/seed/factories/index.js [hunk 1]
```diff
@@ -1,50 +1,67 @@
...
+const relate = (model, type, parameters, options) => {
+  const graphQLClient = new GraphQLClient(seedServerHost, options)
+  const mutation = buildRelationMutation(model, type, parameters)
+  return graphQLClient.request(mutation)
 }
 
 const cleanDatabase = async () => {
   const session = driver.session()
-  const cypher = 'MATCH (n) DETACH DELETE n'
+  const cypher = 'MATCH (n) OPTIONAL MATCH (n)-[r]-() DELETE n,r'
```
Comment (appinteractive) [inline src/seed/factories/index.js]: Why that change?

File: src/seed/reset-db.js [hunk 2]
```diff
@@ -1,30 +1,19 @@
...
 if (process.env.NODE_ENV === 'production') {
-  throw new Error('YOU CAN`T UNSEED IN PRODUCTION MODE')
+  throw new Error(`YOU CAN'T CLEAN THE DATABASE WITH NODE_ENV=${process.env.NODE_ENV}`)
 }
 
-const driver = neo4j().getDriver()
-const session = driver.session()
-
-const deleteAll = `
-MATCH (n)
```
Comment (roschaefer) [inline src/seed/reset-db.js]: @appinteractive here is the former location of the database cleaner query.

File: src/seed/factories/index.js (hunk 1 above)
Comment (appinteractive) [reply src/seed/factories/index.js]: Does the detach not delete also the relation?

File: src/seed/reset-db.js (hunk 2 above)
Comment (appinteractive) [reply src/seed/reset-db.js]: Here the relations or the detach was missing.

**Pull request merged**